script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -426,6 +431,7 @@ They each vary in the extent of their Lifescale,
 [sha:37b6576faaa0fdba9ec1394783ed454ffeac0cae]{
 -- Page 603, Offset 18089.
 -- 西暦以前から活動しているのが『古参』で、
+// Group PR Mark Christian Era doesn't seem correct, Common Era, Anno Domini?
```

Comment:
I guess 'active BC' literally is 'active before christ'. If we don't want jeezy creezy in here we could do
> active before the **common era** are the..

Reply:
I just think Common Era or Anno Domini are more _common_ than Christian Era. I've actually never heard the latter and just assumed it was a lore term during review